packages/cli-core/src/package-manager-controller/package_manager_controller_base.ts

```diff
@@ -52,9 +53,17 @@ export abstract class PackageManagerControllerBase
   /**
    * getWelcomeMessage - returns a welcome message for the customer
    */
-  getWelcomeMessage = () =>
-    `Run \`${this.binaryRunner} amplify help\` for a list of available commands. 
-Get started by running \`${this.binaryRunner} amplify sandbox\`.`;
+  getWelcomeMessage = () => {
+    const welcomeInfo = [
+      `Get started by running ${
+        format.amplifyCommand('sandbox', this.binaryRunner) || ''
```

Comment:
I wonder if an API like `format.runner(this.binaryRunner).amplifyCommand('sandbox')` would be better. Then you can get an amplifyCommand formatter for a specific runner and reuse it like so:
```ts
const { amplifyCommand } = format.runner(this.binaryRunner);
const welcomeInfo = [
  `Get started with ${amplifyCommand('sandbox')}`,
  `Run ${amplifyCommand('help') for a list of available commands`
]
}

```